.envrc

```diff
@@ -52,3 +52,6 @@ fi
 export KKA_AOA_EXCLUDE_PROMETHEUSSTACK="${KKA_AOA_EXCLUDE_PROMETHEUSSTACK:-false}"
 # Dex
 export KKA_AOA_EXCLUDE_DEX="${KKA_AOA_EXCLUDE_DEX:-false}"
+# OpenTelemetry
+export KKA_AOA_EXCLUDE_OPENTELEMETRY_COLLECTOR="${KKA_AOA_EXCLUDE_OPENTELEMETRY_COLLECTOR:-false}"
+export KKA_AOA_EXCLUDE_OPENTELEMETRY_OPERATOR="${KKA_AOA_EXCLUDE_OPENTELEMETRY_OPERATOR:-false}"
```

Comment:
I'd introduce only one env var for OTLM, no need to have two. It's an entire addon. It's all or nothing basically.